smriprep/workflows/outputs.py

```diff
@@ -422,7 +496,9 @@ def _rpt_masks(mask_file, before, after, after_mask=None):
 
 
 def _drop_cohort(in_template):
-    return in_template.split(':')[0]
+    if isinstance(in_template, str):
+        return in_template.split(':')[0]
+    return [_drop_cohort(v) for v in in_template]
```

Comment:
now this is a list of transforms (not using iterables anymore, but using iterfields). because it is a list, I had to extend the implementation.